core/src/main/java/hudson/model/Descriptor.java

```diff
@@ -77,6 +77,7 @@
 import java.util.IdentityHashMap;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
```

Comment:
@stephenc committed to do some bulk rework on this front, but he has not followed up yet. By now all FB annotations annotations are marked as deprecated, and I do not consider using them. IANAL, and my objective is to fix the code